chain/chain/src/store_validator/validate.rs

```diff
@@ -16,6 +16,8 @@ use near_store::{
 
 use crate::{ErrorMessage, StoreValidator};
 
+pub const TOTAL_FUNCTIONS: u64 = 10;
```

Comment:
can you add a comment on what this const is?

Reply:
It will be removed. I've change my mind as all unused functions will be compiled with warnings.